Documentation/building/local-coreclr-corefx.md

```diff
@@ -0,0 +1,29 @@
+# Build Core-Setup with a Local CoreCLR or CoreFX
+
+## Testing with private CoreCLR bits
+
+Generally the Core-Setup build system gets the CoreCLR from a NuGet package which gets pulled down and correctly copied to the various output directories by building `src\pkg\projects\netcoreapp\src\netcoreapp.depproj` which gets built as part of `build.cmd/sh`. For folks that want to do builds and test runs in corefx with a local private build of coreclr you can follow these steps:
+
+1. Build CoreCLR and note your output directory. Ex: `\coreclr\bin\Product\Windows_NT.x64.Release\` Note this will vary based on your OS/Architecture/Flavor and it is generally a good idea to use Release builds for CoreCLR when building Core-Setup and the OS and Architecture should match what you are building in Core-Setup.
+2. Build Core-Setup either passing in the `CoreCLROverridePath` property or setting it as an environment variable:
+
+```batch
+build.cmd /p:CoreCLROverridePath=d:\git\coreclr\bin\Product\Windows_NT.x64.Release\
```

Comment:
Does that trailing `\` work? The other command doesn't have one and the code adds another `\` so I'm guessing this is just out of date. (In my experience another alternative is switching the trailing `\` to `/` for Windows.)

Reply:
Yep this is out of date. I copied this from the CoreFX docs initially.